internal/pkg/service/common/dependencies/mocked.go

```diff
@@ -49,7 +49,14 @@ type mocked struct {
 	bufferStore         *bufferStore.Store
 }
 
-type MockedValues struct {
+type MockedConfig struct {
+	ctx          context.Context
```

Comment:
Zmeny v `MockedConfig`.

Vsade inde to volame `config`, no tu sa to volalo `values`.